tasklist/importer-860/src/main/java/io/camunda/tasklist/zeebeimport/v860/processors/es/ProcessInstanceZeebeRecordProcessorElasticSearch.java

```diff
@@ -74,7 +81,9 @@ public void processProcessInstanceRecord(final Record record, final BulkRequest
         (ProcessInstanceRecordValueImpl) record.getValue();
     if (isVariableScopeType(recordValue) && FLOW_NODE_STATES.contains(record.getIntent().name())) {
       final FlowNodeInstanceEntity flowNodeInstance = createFlowNodeInstance(record);
-      bulkRequest.add(getFlowNodeInstanceQuery(flowNodeInstance));
+      if (isNotProcessType(recordValue) && environment.isTestProfileEnabled()) {
```

Comment:
Yes and no, the importer is not necessary for "production," but it is needed when running the Tasklist ITs. Otherwise, the index remains empty. My current approach is to not refactor the Tasklist ITs; instead, I just import the data for tests. The deletion of the importer code could then happen in a separate step.